python/source/wrappers/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+SET(PyCap_HEADERS
+   ${CMAKE_CURRENT_SOURCE_DIR}/property_tree_wrappers.h
+   ${CMAKE_CURRENT_SOURCE_DIR}/energy_storage_device_wrappers.h
+)
+SET(PyCap_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/property_tree_wrappers.cc
+#    ${CMAKE_CURRENT_SOURCE_DIR}/energy_storage_device_wrappers.cc
```

Comment:
I know but in three months, I will see that line and wondered why it is commented and I will uncomment/delete it :-)